features/locking-rooms.feature

```diff
@@ -18,32 +17,52 @@ Feature: Locking Rooms
   The following scenarios illustrate how these permissions play out
   based upon who is accessing rooms with which access level.
 
-  Scenario: Workspace Admin who knows Room Key enters Locked Room
+  # Wireframe:
+  # https://xd.adobe.com/view/fd425dbe-5384-44c9-997a-eeee6e886a86-a811/screen/04ee266e-931b-4bde-bcf9-af94c7ac444e
+  @wip
+  Scenario: Entering a Locked Room
     Given a Workspace with a Locked Room
-    When a Workspace Admin provides the correct Room Key
-    Then the Workspace Admin is placed in the Room
+    Then a Workspace Member may enter the Room after providing the correct Room Key
+    And a Workspace Member may not enter the Room after providing the wrong Room Key
+    And a Guest may enter the Room after providing the correct Room Key
+    And a Guest may not enter the Room after providing the wrong Room Key
+    # We're not sure if this is actually a good idea, we should check with
+    # design / product to make sure that it makes sense for admins to be able
+    # to barge in like a grumpy parent on prom night.
+    And a Workspace Admin may enter the Room without providing a Room Key
```

Comment:
@maximegalon5 / @colombene  - We were thinking we weren’t sure if a locked room should block out admins or not. What do you think?